Bio/SeqFeature.py

```diff
@@ -851,6 +851,16 @@ def __iter__(self):
             for i in range(self._start, self._end):
                 yield i
 
+    def __eq__(self, other):
+        """Check if start, end and strand of this location are the same as in the other location"""
```

Comment:
Can you include a trailing full stop on the docstrings as per PEP257 please (likewise for the rest of the docstrings, see also #1170).